### PR TITLE
refactor: simplified APR function + unifying AssetValue with internal repo

### PR DESCRIPTION
**Pull request by benesjan**
# Description

1. Simplified `getAPR` function,
2. changed `assetId` type to number,
3. changed `auxData` type to number,
4. importing `AssetValue` from internal repo and doing the related changes,
5. removed `getMarketSize` from element because it was returning non-sense which makes me thing it was not used anyway,
6. changed `interactionNonce` type to number,
7. updated `yarn build` command to not compile typechain twice,
8. removed unused imports and variables from a few places.

# Checklist:

- [x] I have reviewed my diff in github, line by line.
- [x] Every change is related to the PR description.
- [x] There are no unexpected formatting changes, or superfluous debug logs.
- [x] The branch has been rebased against the head of its merge target.
- [x] I'm happy for the PR to be merged at the reviewers next convenience.

### CODE REVIEW

**Comment by benesjan** [comment]
Before merging merge https://github.com/AztecProtocol/aztec-connect-bridges/pull/205 to master and rebase.

**Review by LHerskind — COMMENTED** [review]
The `getMarketSize` is confusingly named and can mean very different things depending on the reader. MarketSize could be understood as similar to market cap, which is very different from liquidity. The current implementations differ in what they use it to describe.

File: src/client/bridge-data.ts [hunk 1]
```diff
@@ -47,11 +43,11 @@ export interface AuxDataConfig {
 export interface BridgeDataFieldGetters {
   /**
    * @dev This function should be implemented for stateful bridges
-   * @param interactionNonce A globally unique identifier of a given DeFi interaction
    * @param inputValue User's input value
+   * @param interactionNonce A globally unique identifier of a given DeFi interaction
```
[the remark this answers is not in the document]
Comment (benesjan) [reply src/client/bridge-data.ts:47]: My brain for some reason thought the order is bad. I have no idea why. Will fix it back.

File: src/client/bridge-data.ts [hunk 2]
```diff
@@ -97,35 +93,22 @@ export interface BridgeDataFieldGetters {
...
-  ): Promise<number[]>;
+   * @notice This function computes annual percentage return (APR) for a given asset
+   * @dev This function should be implemented for all bridges which work with yield-bearing assets
+   * @param yieldAsset A struct detailing the yield asset (e.g. cToken, wstETH, aETH, ...)
+   * @return The expected APR (e.g. for Lido the return value would currently be 3.9)
+   */
+  getAPR?(yieldAsset: AztecAsset): Promise<number>;
 
   /** 
   * @dev This function should be implemented for all bridges dealing with L1 liquidity
```
[the remark this answers is not in the document]
Comment (benesjan) [reply src/client/bridge-data.ts]: Addressed in e0ce8d5b541460306a216acd4afdcf7a48d67295

File: src/client/bridge-data.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (benesjan) [reply src/client/bridge-data.ts]: Addressed in e0ce8d5b541460306a216acd4afdcf7a48d67295

**Review by LHerskind — APPROVED** [review]

**Pull request merged**